spiceaidocs/docs/components/data-connectors/delta-lake.md

```diff
@@ -10,127 +10,106 @@ import TabItem from '@theme/TabItem';
+      name: my_delta_lake_table
     ```
 
-    `spicepod.yaml`
+  </TabItem>
+  <TabItem value="delta_lake_s3" label="Delta Lake + S3" default>
+
     ```yaml
-    version: v1beta1
-    kind: Spicepod
-    name: spice-app
+    - from: delta_lake:s3://my_bucket/path/to/s3/delta/table/  # A reference to a table in S3
+      name: my_delta_lake_table
+      params:
+        delta_lake_aws_region: us-west-2 # Optional
+        delta_lake_aws_access_key_id: ${secrets:aws_access_key_id}
+        delta_lake_aws_secret_access_key: ${secrets:aws_secret_access_key}
+        delta_lake_aws_endpoint: s3.us-west-2.amazonaws.com # Optional
+    ```
 
-    secrets:
-      store: env
+  </TabItem>
+  <TabItem value="delta_lake_azure" label="Delta Lake + Azure Blob">
 
-    # <...>
+    ```yaml
+    - from: delta_lake:abfss://my_container@my_account.dfs.core.windows.net/path/to/azure/delta/table/  # A reference to a table in Azure Blob
+      name: my_delta_lake_table
+      params:
+        # Account Name + Key
+        delta_lake_azure_storage_account_name: my_account
+        delta_lake_azure_storage_account_key: ${secrets:my_key}
+
+        # OR Service Principal + Secret
+        delta_lake_azure_storage_client_id: my_client_id
+        delta_lake_azure_storage_client_secret: ${secrets:my_secret}
+
+        # OR SAS Key
+        delta_lake_azure_storage_sas_key: my_sas_key
     ```
 
-    Learn more about [Env Secret Store](/components/secret-stores/env).
-
   </TabItem>
-  <TabItem value="k8s" label="Kubernetes">
-    ```bash
-    kubectl create secret generic delta_lake \
-      --from-literal=aws-region='<aws-region>'
-      --from-literal=aws-access-key-id='<aws-access-key-id>'
-      --from-literal=aws-secret-access-key='<aws-secret-access-key>'
-    ```
+  <TabItem value="delta_lake_gcp" label="Delta Lake + Google Storage">
 
-    `spicepod.yaml`
     ```yaml
-    version: v1beta1
-    kind: Spicepod
-    name: spice-app
+    params:
+      delta_lake_google_service_account_path: /path/to/service-account.json
+    ```
 
-    secrets:
-      store: kubernetes
+  </TabItem>
+</Tabs>
 
-    # <...>
-    ```
+### Delta Lake object store parameters
 
-    Learn more about [Kubernetes Secret Store](/components/secret-stores/kubernetes).
+Use the [secret replacement syntax](../secret-stores/index.md) to reference a secret, e.g. `${secrets:aws_access_key_id}`.
 
-  </TabItem>
-  <TabItem value="keyring" label="Keyring">
-    Add new keychain entry (macOS), with secrets in JSON string
+#### AWS S3
 
-    ```bash
-    security add-generic-password -l "Delta Lake Secret" \
-    -a spiced -s spice_secret_delta_lake \
-    -w $(echo -n '{"aws-region": "<aws-region>", "aws-access-key-id": "<aws-access-key-id>", "aws-secret-access-key": "<aws-secret-access-key>"}')
-    ```
+- `databricks_aws_region`: Optional. The AWS region for the S3 object store. E.g. `us-west-2`.
```

Comment:
Should these be databricks_ or delta_lake_?

Reply:
Changed to `delta_lake`